pkg/network/protocols/postgres/model_linux.go

```diff
@@ -60,29 +61,27 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
-var (
-	regexMapper = map[Operation]*regexp.Regexp{
-		CreateTableOP: regexp.MustCompile(`CREATE TABLE (\S+)`),
-		InsertOP:      regexp.MustCompile(`INSERT INTO (\S+)`),
-		DropTableOP:   regexp.MustCompile(`DROP TABLE(\s*IF\s+EXISTS\s+)?(\S+)`),
-		UpdateOP:      regexp.MustCompile(`UPDATE (\S+)`),
-		SelectOP:      regexp.MustCompile(`SELECT .* FROM (\S+)`),
-	}
-)
-
 // extractTableName extracts the table name from the query.
 func (e *EventWrapper) extractTableName() string {
```

Comment:
Let's create benchmarks reports for the new and the old version

Reply:
Done:
Old: BenchmarkExtractTableName-4   	 4783006	       244.3 ns/op	      64 B/op	       3 allocs/op - before the change
New: BenchmarkExtractTableName-4   	 1592484	       669.6 ns/op	     408 B/op	      14 allocs/op

Reply:
After the latest change the results are:
BenchmarkExtractTableName-4   	 2099086      556.0 ns/op	     264 B/op	      11 allocs/op